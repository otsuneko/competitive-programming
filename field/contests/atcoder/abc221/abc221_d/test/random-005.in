9
7 2
5 2
3 4
9 5
4 2
6 1
1 9
9 10
1 7
