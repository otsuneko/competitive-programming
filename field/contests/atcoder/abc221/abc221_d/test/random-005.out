5 7 3 1 2 0 0 0 0
